platform/admin/admin-configuration-configupdater/src/main/java/org/codice/ddf/admin/configuration/ConfigurationUpdater.java

```diff
@@ -143,8 +140,8 @@ public void handleStore(ConfigurationContext context) throws IOException {
                 return null;
               });
     } catch (PrivilegedActionException e) {
-      // Only caught if the action through a CHECKED exception, thus it MUST be an IOException.
-      throw new IOException(e);
+      // Only caught if the action threw a CHECKED exception, thus it MUST be an IOException.
+      throw (IOException) e.getCause();
```

Comment:
✏️ It's belt & suspenders as we _know_ what the checked exception is; however, you should affirm that it is `instanceof` prior to your cast. In the else block (where you'll never actually end up), you can throw a `new IOException(e.getCause())`

Reply:
Just did a check with SonarLint and this does not appear as an issue. Do we really need to add that verbosity?

Reply:
It is not a matter of verbosity but more an API matter. the cause can be anything and to be safe, you should check its instance of before blindly type casting it